dashboardsSanitizer/src/test/java/org/opensearch/migrations/dashboards/SanitizerTest.java

```diff
@@ -0,0 +1,6 @@
+package org.opensearch.migrations.dashboards;
+
+import static org.junit.jupiter.api.Assertions.*;
+class SanitizerTest {
```

Comment:
Lets add some test cases to covers the key scenarios for objects that are compatible and transformed along with a couple of cases where they cannot be transformed.